back_end/front_end/src/components/navigation/Navigation.js

```diff
@@ -1,13 +1,22 @@
 import React from "react";
-
-import { NavLink } from "react-router-dom";
+import home from "./../../images/home.svg";
+import { useHistory } from "react-router-dom";
+import { IconButton } from "@material-ui/core";
+import "./Navigation.css";
 
 const Navigation = () => {
+  let history = useHistory();
   return (
     <div>
-      <NavLink to="/">Home</NavLink>
-      <NavLink to="/CAN">CAN</NavLink>
-      <NavLink to="/settings">Settings</NavLink>
+      <IconButton color="primary" aria-label="home view" href="/">
+        <img src={home} className="App-logo" alt="home" />
+      </IconButton>
+      <button class="button button1" onClick={() => history.goBack()}>
```

Comment:
Would be better to name these buttons more appropriately, like back/forward button. Also, if I hit the back button from the home page, it takes me out of the app. As in, it takes me to the blank page that I see when I open the browser first. So basically if we are in home, it should no longer allow going back any more.

Reply:
That's a good point, I switched it to back-button and forward-button, although these will buttons will get removed in a future JIRA when we switch to swiping https://github.com/TejM/DashPi/pull/10/commits/f9281568ccd3ba015bb6e596b8e8da930fe8f1aa

Also I see what mean about how you can go backwards, but that's if you manually type in localhost:3000 from a blank tab. It's not a workflow that the DashPi will ever go through because it auto boots the first page as home page which would make the furthest back you can go on a page is the boot up home page

Reply:
I actually would like to try tackling the swipe functionality for my next JIRA, https://the-disk-golfers.atlassian.net/browse/DAS-31